packages/sdk/src/Confidence.test.ts

```diff
@@ -2,76 +2,76 @@ import { Confidence } from './Confidence';
 
 describe('Confidence', () => {
   describe('context', () => {
-    it('returns immutable values', () => {
+    it('returns immutable values', async () => {
       const confidence = new Confidence({} as any);
-      const context = confidence.getContext();
+      const context = await confidence.getContext();
```

Comment:
All these awaits added in this file should be removed.

Reply:
fixed.